### PR TITLE
[DO NOT MERGE] Update to MSAL 3.0.4

### PR DESCRIPTION
**Pull request by bgavrilMS**

### CODE REVIEW

File: active-directory-dotnet-native-uwp-v2/active-directory-dotnet-native-uwp-v2.csproj [hunk 1]
```diff
@@ -126,10 +126,10 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Identity.Client">
-      <Version>3.0.3-preview</Version>
+      <Version>3.0.4-preview</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
-      <Version>6.1.9</Version>
+      <Version>6.2.8</Version>
```
Comment (jmprieur) [inline active-directory-dotnet-native-uwp-v2/active-directory-dotnet-native-uwp-v2.csproj:132]: Is this necessary? (that's ok, BTW)

File: active-directory-dotnet-native-uwp-v2/MainPage.xaml.cs [hunk 2]
```diff
@@ -75,7 +75,7 @@ private async void CallGraphButton_Click(object sender, RoutedEventArgs e)
 
                 try
                 {
-                    authResult = await PublicClientApp.AcquireTokenInteractive(scopes, this)
+                    authResult = await PublicClientApp.AcquireTokenInteractive(scopes)
```
Comment (jmprieur) [inline active-directory-dotnet-native-uwp-v2/MainPage.xaml.cs:78]: Don't we want to add .WithParentActivityOrWindow?

**Review by jmprieur — COMMENTED** [review]
LGTM
I've asked 2 questions

File: active-directory-dotnet-native-uwp-v2/active-directory-dotnet-native-uwp-v2.csproj (hunk 1 above)
Comment (bgavrilMS) [reply active-directory-dotnet-native-uwp-v2/active-directory-dotnet-native-uwp-v2.csproj:132]: No, it is not required, but I like it when samples are up to date with the latest dependencies. Looks clean.

File: active-directory-dotnet-native-uwp-v2/MainPage.xaml.cs (hunk 2 above)
Comment (bgavrilMS) [reply active-directory-dotnet-native-uwp-v2/MainPage.xaml.cs:78]: There isn't any on UWP.

**Pull request merged**